### DISCUSSION TITLE
`outline-style: auto` support

### DISCUSSION THREAD

**Discussion by n1ckoates** [tailwindlabs/tailwindcss#10321] [Ideas]
Tailwind doesn't support [`outline-style: auto`](https://developer.mozilla.org/en-US/docs/Web/CSS/outline-style#auto), which is useful for showing a "native" browser focus ring. Support for this under `outline-auto` would be awesome, and I imagine it shouldn't be too complex to add.

**Comment by hugo-cardoso** [comment]
Up vote

**Comment by Knowles5747** [comment]
I understand that you are looking for support for outline-style: auto in Tailwind CSS. [According to a discussion on GitHub, it seems that this feature is not currently supported by Tailwind CSS ](https://github.com/tailwindlabs/tailwindcss/discussions/10321)[1](https://github.com/tailwindlabs/tailwindcss/discussions/10321). [However, you can use the outline-none utility to hide the default browser outline on focused elements and apply your own focus styling for accessibility ](https://tailwindcss.com/docs/outline-style)[2](https://tailwindcss.com/docs/outline-style). [american honda finance](https://www.hondafinance.online/)
